build.sbt

```diff
@@ -23,7 +23,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "2.0.4",
+  version := "2.0.5-SNAPSHOT",
```

Comment:
Anything new in here we want?

Reply:
not really just I told @emelvarol-cognite that I will improve the test for DMI a bit, this can go in the next release we don't have to bump version for now 😅